.evergreen/config.yml

```diff
@@ -0,0 +1,47 @@
+functions:
+  fetch source:
+    # Executes git clone
+    - command: git.get_project
+      params:
+        directory: src
+    # Applies the submitted patch, if any
+    - command: git.apply_patch
+
+  exec script:
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src
+        shell: bash
+        script: |
+          ${PREPARE_SHELL}
+          ${file}
+
+pre:
+  - func: fetch source
+
+post:
+
+tasks:
+  - name: static-check
+    tags: ["ci"]
+    commands:
+      - func: exec script
+        vars:
+          file: .evergreen/static-check.sh
+
+  - name: run-test
+    tags: ["ci"]
+    commands:
+      - func: exec script
+        vars:
+          file: .evergreen/run-test.sh
+
+buildvariants:
+
+  - name: simple-variant
+    tags: ["ci"]
+    display_name: Simple Variant
```

Comment:
Do we want all of these to have the same display name?  In Java driver, for example, the display name for static checks is "Static Checks".

Reply:
This display_name is for variant, not for task, so its counterpart is like: 

**4.2 Sharded Cluster NoAuth NoSSL JDK8 Linux**

Reply:
I changed it to concrete variant name instead. We'd better avoid ambiguous 'simple-variant' expression.